cli/tools/test_runner.rs

```diff
@@ -63,42 +108,295 @@ pub fn prepare_test_modules_urls(
   Ok(prepared)
 }
 
-pub fn render_test_file(
-  modules: Vec<Url>,
+pub async fn run_test(
```

Comment:
I think the name of this function is confusing, does it run only a single test case from a file?

Reply:
Renamed to run_test_file

Reply:
Thanks